src/Notification/Notification.js

```diff
@@ -0,0 +1,287 @@
+/* @flow */
+
+import * as React from 'react';
+import { Animated } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import { getStatusBarHeight } from 'react-native-status-bar-height';
+
+import StyleSheet from '../PlatformStyleSheet';
+import Alert from './Alert';
+import Warning from './Warning';
+
+import type {
+  OnLayout,
+  NotificationStyleType,
+  NotificationType,
+} from '../types';
+import type { StylePropType } from '../PlatformStyleSheet/StyleTypes';
+
+type Props = {|
+  +style?: StylePropType,
+  +notificationType: NotificationType,
+  +onDismiss?: () => void,
+|};
+
+type State = {|
+  layout: {
+    height: number,
+    measured: boolean,
+  },
+  opacity: Animated.Value,
+  translateY: Animated.Value,
+  isOpen: boolean,
+  notificationStyle: NotificationStyleType,
+  notificationTitle: React.Node | string,
+  notificationMessage: React.Node | string,
+|};
+
+const ANIMATION_DURATION = 250;
+const DISPLAY_DURATION = 1500;
+
+export default class Notification extends React.Component<Props, State> {
+  state = {
+    layout: {
+      height: -100,
+      measured: false,
+    },
+    opacity: new Animated.Value(0),
+    translateY: new Animated.Value(0),
+    isOpen: false,
+    notificationStyle: 'error',
+    notificationTitle: '',
+    notificationMessage: '',
+  };
+
+  _lastCallTimestamp = null;
```

Comment:
Is there a specific reason to use underscores? I think we should add this as an eslint rule to prevent them. What do you think?

Reply:
In that file I have two `toggleAlert` functions where one is internal (with `underscore) and second is external. I just add these `underscores` to keep the convention. I'm also not a fan of using underscores, but in that case it was the best idea imo.

Reply:
Maybe a way to make it very explicit would be to replace the underscore with `internal`. I don't have a strong opinion about it.